README.md

```diff
@@ -38,6 +38,7 @@ export const buyTx = async (): Promise<void> => {
   const token = await Token.create({
     tokenAddress: '0x1234567890123456789012345678901234567890',
     moonshot,
+    provider,
```

Comment:
don't you think it's not abstracted enough if you ask the dev to pass in a provider ? I think the provider should be initialized within the SDK

Reply:
I did not really make a change here, Token class requires a `provider` to be passed as an argument and the example in readme is incorrect. 

I can make a change to simplify this, but let me do it in another PR.

Reply:
good point, let's wait with this to not make any breaking changes